src/config/firebase.prod.json

```diff
@@ -0,0 +1,8 @@
+{
+    "apiKey": "AIzaSyBXuACowZcLcr1wlhM53LHtibFwa59EmAY",
```

Comment:
This is supposed to be public, right?

Reply:
Yeah, it's a key meant for client-side distribution.